x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -53,7 +53,7 @@ export const policyFactory = (
       },
       behavior_protection: {
         mode: ProtectionModes.prevent,
-        reputation_service: false,
+        reputation_service: cloud, // Defaults to true if on cloud
```

Comment:
are you sure we should fix boolean with other values ('cloud', but is there any other options too?) ?